packages/stores/src/account/cosmos.ts

```diff
@@ -432,6 +435,20 @@ export class CosmosAccountImpl {
       signOptions
     );
 
+    const getPublicKeyTypeUrl = (coinType: number, chainId: string): string => {
+      if (coinType === 60) {
```

Comment:
This should use the `useEthereumSign` field rather than check the `coinType` since this is part of the signature generation.

Reply:
Otherwise, looks fine to the Evmos team.

Reply:
Updated, thanks for the feedback!